backend/Makefile

```diff
@@ -63,12 +64,21 @@ image_launcher:
 
 .PHONY: dev-kind-cluster
 dev-kind-cluster:
+	${CONTAINER_ENGINE} build -t ${IMG_TAG_WEBHOOK_PROXY} -f $(CURDIR)/../tools/kind/Dockerfile.webhook-proxy $(CURDIR)/../tools/kind
 	-kind create cluster --name $(KIND_NAME) --config $(CURDIR)/../tools/kind/kind-config.yaml
 	kubectl config use-context kind-$(KIND_NAME)
 	kind get kubeconfig --name $(KIND_NAME) > $(CURDIR)/../kubeconfig_$(KIND_NAME)
+	kind --name $(KIND_NAME) load docker-image ${IMG_TAG_WEBHOOK_PROXY}
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.16.2/cert-manager.yaml
```

Comment:
This is a different version of cert-manager than in the GitHub workflows. Perhaps we could have a Make target that installs cert-manager and this could be reused here and in the GitHub workflows.